tests/conftest.py

```diff
@@ -0,0 +1,66 @@
+"""
+Common and builtin fixtures
+"""
+import pytest
+from unittest.mock import MagicMock
+
+import epsagon
+
+TEST_TOKEN = 'test'
+TEST_APP = 'test_app'
+TEST_COLLECTOR = 'collector'
+
+
+class TestTransport(MagicMock):
+    """
+    Mock trace transport for tests
+    """
+
+    @property
+    def last_trace(self):
+        """
+        :return: The last Trace object that was sent (None if no trace was sent)
+        """
+        return self.send.call_args[0][0] if self.send.call_args else None
+
+    @property
+    def sent_traces(self):
+        """
+        :return: List of all the Trace objects that were sent
+        """
+        return [
+            call_args[0][0] for call_args in self.send.call_args_list
+        ]
+
+
+@pytest.fixture(scope='function', autouse=False)
```

Comment:
I wanted to use `autouse=True` but many tests in `test_trace.py` are checking the HTTP request